corehq/apps/app_manager/templates/app_manager/v2/partials/releases.html

```diff
@@ -12,9 +12,21 @@
        });
     });
     $(function () {
+        hqImport('hqwebapp/js/urllib.js').registerUrl(
+            'form_data',
+            '/a/' + '{{ domain }}' + '/apps/view/---/form_data/'
```

Comment:
Surprised to see a url built like this instead of using `registerurl` in a template. Although I think releases.js does some funky stuff with urls, is that why this is like this?